src/components/course/data/utils.jsx

```diff
@@ -16,7 +16,7 @@ export function getActiveCourseRun(course) {
   if (!course || !course.courseRuns || course.courseRuns.length === 0) {
     return undefined;
   }
-  return course.courseRuns.pop();
+  return course.courseRuns[course.courseRuns.length - 1];
```

Comment:
.pop() mutates the original array, swapping this to just return the last element